3.4/deployment-cloud-aws.md

```diff
@@ -1,25 +1,32 @@
 ---
 layout: default
-description: ArangoDB can be deployed on AWS or other cloud platforms
+description: ArangoDB can be deployed on Amazon Web Services (AWS) in multiple ways.
 ---
 Deploying ArangoDB on AWS
 =========================
 
-ArangoDB can be deployed on AWS or other cloud platforms. Deploying on a cloud
-provider is common choice and many of the most big ArangoDB installation are running
-on the cloud.
+ArangoDB can be deployed on AWS or other cloud platforms. AWS is the
+infrastructure provider choice for some of the largest ArangoDB installations.
 
-Up to ArangoDB 3.2, official ArangoDB AMI were available in the [AWS marketplace](https://aws.amazon.com/marketplace/search/results/ref=dtl_navgno_search_box?page=1&searchTerms=arangodb){:target="_blank"}.
+Up to and including ArangoDB 3.2, official ArangoDB AMI were available in the
+[AWS marketplace](https://aws.amazon.com/marketplace/search/results/ref=dtl_navgno_search_box?page=1&searchTerms=arangodb){:target="_blank"}.
 Such AMI are not being maintained anymore, though. However, deploying on AWS is
 still possible, and again, a quite common scenario.
 
 After having initialized your preferred AWS instance, with one of the ArangoDB supported
 operating systems, using the [ArangoDB Starter](deployment-arango-dbstarter.html),
 performing a [Manual Deployment](deployment-manually.html),
-or using [Kubernetes](deployment-kubernetes.html), 
-are all valid options to deploy on AWS. Please refer to the corresponding chapters for further 
-information.
+or using [Kubernetes](deployment-kubernetes.html),
+are all valid options to deploy on AWS. Please refer to the corresponding
+chapters for further information.
```

Comment:
Can we add a link?

Reply:
This seems to refer to the three links just above, therefore I removed this sentence as it doesn't add any value.